### PR TITLE
Add exec command

### PR DESCRIPTION
**Pull request by tonistiigi**
Starts a new process inside an existing container.

Signed-off-by: Tonis Tiigi <tonistiigi@gmail.com>

### CODE REVIEW

File: exec.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	handler := newSignalHandler(tty)
+	defer handler.Close()
+	if err := container.Start(process); err != nil {
+		return -1, err
+	}
+
+	return handler.forward(process)
+}
+
+// loadProcessConfig loads the process configuratio from the provided path.
```
Comment (mrunalp) [inline exec.go]: typo: configuratio => configuration

File: exec.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+	"github.com/opencontainers/specs"
+)
+
+var execCommand = cli.Command{
+	Name:  "exec",
+	Usage: "execute new process inside the container",
+	Action: func(context *cli.Context) {
+
```
Comment (crosbymichael) [inline exec.go]: can you remove all the whitespace ;) plz?

Comment (tonistiigi) [inline exec.go]: sure :)

**Comment by laijs** [comment]
I prefer the name ```enter```, the ```exec```  is ambiguous

**Comment by wking** [comment]
On Wed, Aug 19, 2015 at 05:34:00PM -0700, Lai Jiangshan wrote:
> I prefer the name ```enter```, the ```exec```  is ambiguous

There was some discussion of the name for this functionality in #210
as well.  I'd mentioned ‘nsenter’ [1] and ‘ip netns exec’ [2] (as well
as ‘docker exec’), so it could go either way without being too
surprising.  But @duglin [3] and I [4] currently favor ‘exec’ over
‘enter’.

[1]: https://github.com/opencontainers/runc/pull/210#issuecomment-132269794
[2]: https://github.com/opencontainers/runc/pull/210#issuecomment-132271098
[3]: https://github.com/opencontainers/runc/pull/210#issuecomment-132276297
[4]: https://github.com/opencontainers/runc/pull/210#issuecomment-132280234

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**